cheerio-scraper/INPUT_SCHEMA.json

```diff
@@ -117,6 +117,13 @@
             "maximum": 360,
             "unit": "secs"
         },
+        "additionalMineTypes": {
```

Comment:
I'd would consider to implement this as type ARRAY with JAVASCRIPT editor, it's not so user-friendly, but this way we will be able to migrate to multi-select which will store underlying content as an array